packages/jest-core/src/watch.ts

```diff
@@ -51,30 +56,34 @@ const INTERNAL_PLUGINS = [
   QuitPlugin,
 ];
 
-const RESERVED_KEY_PLUGINS = new Map([
+// TODO: Is it correct with constructor here?
```

Comment:
Hey @tdd 👋 Does this make sense to you? 🙂

Reply:
It shouldn't 😅 Tests fail

![image](https://user-images.githubusercontent.com/1404810/53492368-f87ea480-3a98-11e9-83f6-60aa9c6adb14.png)

Any good idea on how to type it properly? I went with `Function` for now, but it should really be narrower